src/app.js

```diff
@@ -0,0 +1,13 @@
+import express from "express"
+import bodyParser from "body-parser";
+// import router from "./routes/index"
+
+import routes from "./routes/index";
+import router from "./routes/index";
+const app=express();
+app.use(express.json());
+app.use(bodyParser.json());
+app.use('/task',router)
+const port=process.env.PORT || 3000;
+app.listen(port,()=>console.log(`server is running at ${port}`));
+export default app;
```

Comment:
Remove this line it has nothing to do in this file

Reply:
Done